test/dependencies/go.mod

```diff
@@ -17,13 +17,13 @@ require (
 	github.com/ipfs/go-unixfs v0.2.4
 	github.com/ipfs/hang-fds v0.0.2
 	github.com/ipfs/iptb v1.4.0
-	github.com/ipfs/iptb-plugins v0.2.2
+	github.com/ipfs/iptb-plugins v0.2.3-0.20200429202754-744595369b71
 	github.com/ipld/go-ipld-prime v0.0.2-0.20191108012745-28a82f04c785
 	github.com/jbenet/go-random v0.0.0-20190219211222-123a90aedc0c
 	github.com/jbenet/go-random-files v0.0.0-20190219210431-31b3f20ebded
 	github.com/libp2p/go-libp2p v0.5.2
-	github.com/libp2p/go-libp2p-core v0.3.1
-	github.com/multiformats/go-multiaddr v0.2.0
+	github.com/libp2p/go-libp2p-core v0.5.2
```

Comment:
I'd upgrade all deps here:

```go
go get $(go list -json -m all | jq -r 'select(.Indirect != true and .Main != true) | "\(.Path)@latest"')
go mod tidy
```

Reply:
leads to
```
github.com/ipfs/go-ipfs/test/dependencies/graphsync-get imports
	github.com/ipld/go-ipld-prime/impl/free: module github.com/ipld/go-ipld-prime@latest found (v0.0.3), but does not contain package github.com/ipld/go-ipld-prime/impl/free
```

should i cut a release of iptb-plugins? otherwise latest won't pick up the master commit i think

Reply:
Ah, you'll probably need to downgrade go-ipld-prime. But yeah, feel free to cut a release of iptb-plugins.